api/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ All notable changes to the **Prowler API** are documented in this file.
 ### Changed
 - Now the MANAGE_ACCOUNT permission is required to modify or read user permissions instead of MANAGE_USERS [(#8281)](https://github.com/prowler-cloud/prowler/pull/8281)
 - Now at least one user with MANAGE_ACCOUNT permission is required in the tenant [(#8729)](https://github.com/prowler-cloud/prowler/pull/8729)
+- Enhance deprecation warning for M365 user and password authentication [(#8913)](https://github.com/prowler-cloud/prowler/pull/8913)
```

Comment:
```suggestion
```

This does not require a changelog entry since it's refining the documentation after a PR that has not been released.